tests/test-addons.py

```diff
@@ -231,7 +268,6 @@ def test_observability(self):
         validate_observability()
         print("Disabling observability")
         microk8s_disable("observability")
-        microk8s_reset()
```

Comment:
Why was this removed? I thought the reset is made a no-op in multinode tests.

Reply:
In the `clean_up()` fixture we check the setup. If it's a multinode, we skip `microk8s_reset` other wise we execute it at the end of every test.